Codeinwp/neve
# Menu icon adjustments / dropdown icon [#1157]

### Summary
Makes close button from sidebar inherit style from the menu icon component.
Change dropdown CSS chevron to a SVG icon.


### Will affect visual aspect of the product

YES

### Test instructions


- Check that the Menu Icon settings apply to the close sidebar button.
- Check that the new dropdown arrows work well 


Closes #1157.
